Traceback (most recent call last):
  File "C:\Users\jqbon\anaconda3\Lib\site-packages\jupyter_cache\executors\utils.py", line 58, in single_nb_execution
    executenb(
  File "C:\Users\jqbon\anaconda3\Lib\site-packages\nbclient\client.py", line 1305, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\jqbon\anaconda3\Lib\site-packages\jupyter_core\utils\__init__.py", line 165, in wrapped
    return loop.run_until_complete(inner)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\jqbon\anaconda3\Lib\asyncio\base_events.py", line 687, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "C:\Users\jqbon\anaconda3\Lib\site-packages\nbclient\client.py", line 705, in async_execute
    await self.async_execute_cell(
  File "C:\Users\jqbon\anaconda3\Lib\site-packages\nbclient\client.py", line 1058, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "C:\Users\jqbon\anaconda3\Lib\site-packages\nbclient\client.py", line 914, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
###############################################################
# This block loads the various packages used in this notebook #
###############################################################

using DelimitedFiles #read and write to .csv files 
using Printf         #formatted f strings for printing
using Polynomials    #package features basic linear regression tools (among other things)
using CairoMakie     #package for generating publication-quality figures
using LaTeXStrings   #this package provides support for LaTeX syntax in strings; useful for formatting
using JLD2           #saves and loads julia data structures
using DataFrames     #create data frames
using PrettyTables   #this package provides nice tabular formatting for display in notebook
------------------


Error opening package file C:\Users\jqbon\.julia\compiled\v1.11\MathTeXEngine\eCCtS_yWaUM_1.dll: The paging file is too small for this operation to complete. 


Stacktrace:
  [1] _include_from_serialized(pkg::Base.PkgId, path::String, ocachepath::String, depmods::Vector{Any}, ignore_native::Nothing; register::Bool)
    @ Base .\loading.jl:1245
  [2] _include_from_serialized (repeats 2 times)
    @ .\loading.jl:1210 [inlined]
  [3] _require_search_from_serialized(pkg::Base.PkgId, sourcepath::String, build_id::UInt128, stalecheck::Bool; reasons::Dict{String, Int64}, DEPOT_PATH::Vector{String})
    @ Base .\loading.jl:2041
  [4] _require(pkg::Base.PkgId, env::String)
    @ Base .\loading.jl:2527
  [5] __require_prelocked(uuidkey::Base.PkgId, env::String)
    @ Base .\loading.jl:2388
  [6] #invoke_in_world#3
    @ .\essentials.jl:1089 [inlined]
  [7] invoke_in_world
    @ .\essentials.jl:1086 [inlined]
  [8] _require_prelocked(uuidkey::Base.PkgId, env::String)
    @ Base .\loading.jl:2375
  [9] macro expansion
    @ .\loading.jl:2314 [inlined]
 [10] macro expansion
    @ .\lock.jl:273 [inlined]
 [11] __require(into::Module, mod::Symbol)
    @ Base .\loading.jl:2271
 [12] #invoke_in_world#3
    @ .\essentials.jl:1089 [inlined]
 [13] invoke_in_world
    @ .\essentials.jl:1086 [inlined]
 [14] require(into::Module, mod::Symbol)
    @ Base .\loading.jl:2260

